src/components/checkbox/checkbox.js

```diff
@@ -96,10 +97,11 @@ class StatelessCheckbox extends React.Component<PropsT, StatelessStateT> {
   };
 
   render() {
+    const {checkMarkType} = this.props;
```

Comment:
Thoughts on just making this a boolean prop called `toggle`?

<Checkbox toggle/>

Seems like a slightly simpler API for end users

Reply:
there are plenty of checkbox styles. Just because we have only two so far doesn't make it boolean